src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,90 @@
 import React from 'react';
+import cn from 'classnames';
+import { useTodos } from '../../utils/TodosContext';
+import { Filter } from '../../types/Context';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+export const TodoList: React.FC = () => {
+  const { todos, filter, query, selectedTodoId, setSelectedTodoId } =
+    useTodos();
```

Comment:
`TodoList` shouldn't have any logic like filtering, just accept todos as a prop and render them